packages/components/pie-card/test/component/pie-card.spec.ts

```diff
@@ -304,4 +304,31 @@ test.describe('PieCard - Component tests', () => {
             });
         });
     });
+
+    test.describe('Prop: disabled', () => {
+        test.describe('when an image exists and the prop `disabled` is set to `true`', () => {
+            test('should set the opacity to 50%', async ({ mount, page }) => {
```

Comment:
I think this probably needs a visual test too?

Reply:
Added.